src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -142,42 +140,26 @@ private StateUpdate ToStateUpdate(Delta delta)
 
         private Address GetAccountAddress(ByteString publicKeyByteString)
         {
-            if (publicKeyByteString == null || publicKeyByteString.IsEmpty)
-            {
-                return null;
-            }
+            if (publicKeyByteString == null || publicKeyByteString.IsEmpty) return null;
 
-            IPublicKey publicKey = _cryptoContext.GetPublicKeyFromBytes(publicKeyByteString.ToByteArray());
+            var publicKey = _cryptoContext.GetPublicKeyFromBytes(publicKeyByteString.ToByteArray());
             return publicKey.ToKvmAddress();
         }
 
-        private static void QuickFail(ContractEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
+        private static void QuickFail(PublicEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
         {
             // here we need to propagate back to Delta
             env.CurrentBlock.GasUsed += (long) entry.GasLimit;
             if (txTracer.IsTracingReceipt)
-            {
                 txTracer.MarkAsFailed(env.ExecutingAccount, (long) entry.GasLimit, Bytes.Empty, "invalid");
-            }
         }
 
         private void Execute(Delta delta, ITxTracer txTracer, bool readOnly)
         {
             var stateUpdate = ToStateUpdate(delta);
-            foreach (var publicEntry in delta.PublicEntries)
-            {
-                var contractEntry = new ContractEntry
-                {
-                    Base = publicEntry.Base, Amount = publicEntry.Amount, Data = ByteString.Empty, GasLimit = 21000
-                };
-                Execute(contractEntry, stateUpdate, txTracer, readOnly);
-            }
 
             // revert state if any fails (take snapshot)
-            foreach (var contractEntry in delta.ContractEntries)
-            {
-                Execute(contractEntry, stateUpdate, txTracer, readOnly);
-            }
+            foreach (var publicEntry in delta.PublicEntries) Execute(publicEntry, stateUpdate, txTracer, readOnly);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)